src/openfermion/chem/molecular_data.py

```diff
@@ -324,6 +324,7 @@ def geometry_from_file(file_name):
 def antisymtei(two_body_integrals):
     """
     Args:
+
```

Comment:
Is this correct formatting?  Usually its Args: followed by an indented argument, no line break.  (I think)